core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -231,6 +243,12 @@ public void setMaster(@Nullable SchemaRegistryIdentity newMaster)
         masterRestService = null;
       } else {
         masterRestService = new RestService(masterIdentity.getUrl());
+        if ("https".equals(masterIdentity.getScheme()) && !masterRestSslInitialized) {
```

Comment:
i know it didn't happen in previous security prs, but we should probably get `"http"` and `"https"` as constants somewhere as they are proliferating quite a bit and a typo wouldn't be caught by the compiler